app/controllers/application_controller.rb

```diff
@@ -703,20 +692,16 @@ def build_audit_msg(new, current, msg_in)
           current[k] = {} if !current.nil? && current[k].nil?
           #   process keys of the current and new hashes
           (new[k].keys | (current.nil? ? [] : current[k].keys)).each do |hk|
-            if current.nil? || (new[k][hk] != current[k][hk])
-              if password_field?(hk) # Asterisk out password fields
-                msg_arr << "#{hk}:[*]#{' to [*]' unless current.nil?}"
-              else
-                msg_arr << "#{hk}:[" +
-                  (current.nil? ? "" : "#{filter_config(current[k][hk])}] to [") +
-                  "#{filter_config(new[k][hk])}]"
-              end
-            end
+            next if current.present? && (new[k][hk] == current[k][hk])
+            msg_arr << if password_field?(hk) # Asterisk out password fields
+                         "#{hk}:[*]#{' to [*]' unless current.nil?}"
```

Comment:
unless used *inside* a string interpolation :nauseated_face: 

Probably a separate PR, right?

```
$ ag '"[^"]*#{[^}]*\b(unless|if)'
app/controllers/application_controller.rb
700:          msg_arr << "#{k}:[*]#{' to [*]' unless current.nil?}"
708:                msg_arr << "#{hk}:[*]#{' to [*]' unless current.nil?}"

app/controllers/report_controller/reports/editor.rb
811:            (suffix ? " (#{MiqReport.date_time_break_suffixes.collect { |s| s.first i  s.last == suffix }.compact.join})" : "") +

app/helpers/vm_helper/textual_summary.rb
259:         :value => "#{s.name}#{" (main)" if s == main}",
```

Reply:
Yes, I'd rather address that in a separate PR.

Reply:
Here: https://github.com/ManageIQ/manageiq-ui-classic/pull/4671